KenticoCloudDotNetGenerators.Tests/CodeGeneratorTests.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Microsoft.CodeAnalysis;
+using NUnit.Framework;
+using System.IO;
+using System.Linq;
+
+namespace KenticoCloudDotNetGenerators.Tests
+{
+    [TestFixture]
+    public class CodeGeneratorTests
+    {
+        [TestCase]
+        public void IntegrationTest()
```

Comment:
right. would it make sense to point it to the same project we reference from the SDK? (or apiary)?